features/clamav/test/test_clamav_cron.py

```diff
@@ -0,0 +1,8 @@
+from helper.utils import execute_remote_command
+
+
+def test_clamav_cron(client):
+    # This will already fail if file is absent
+    out = execute_remote_command(client, "cat /var/spool/cron/crontabs/root")
```

Comment:
If this file fails, we will get an error message like this then:
```
cat: /var/spool/cron/crontabs/root: No such file or directory\n
```
Can we adjust the error message in order to make it more clear what the issue is about? Maybe something like this:
```
Cron configuration for ClamAV expected but not found.
```

Reply:
Sure we can but I think when it comes to debugging for someone who isn't into PyTest and Garden Linux, the first error message (even it's not catched) makes more sense. It directly tells the user what, where and why there's an issue while your requested change would lead into more questions. Why did it fail, what it the issue and where and how to debug this?

Reply:
Is it? I mean if you not into pytest or Garden Linux, you don't really know why a missing `/var/spool/cron/crontabs/root` is a problem. You need then to check what the test is doing and parse Python code. For me at least, it is more confusing if I get a `cat: /var/spool/cron/crontabs/root: No such file or directory\n` error message instead of a error message which tries to explain to me, why this missing file is a problem in this context. This should be more user friendly imho.

Reply:
Unfortunately, I don't share your opinion here. Let's solve it by using both - your output together with the error message 👌